libraries/lib-datahandler/src/main/thrift/users.thrift

```diff
@@ -54,6 +54,7 @@ struct User {
     9: optional string lastname, // lastname or surname of the person
     10: required string department,
     11: optional bool wantsMailNotification,
+    12: optional string commentMadeDuringModerationRequest,
```

Comment:
I don't like having this field in the `User` structure. Why did you prefer using the User to pass the comment over adding a comment parameter to the respective methods?